examples/rbac/client/feature_store.yaml

```diff
@@ -0,0 +1,14 @@
+project: server
+registry:
+    registry_type: remote
+    path: localhost:6570
+offline_store:
+    type: remote
+    host: localhost
+    port: 8815
+online_store:
+    type: remote
+    path: http://localhost:6566
+auth:
+    type: kubernetes
```

Comment:
is this client setting good for auth ?
How this generate the token ?

Reply:
Yes, this gets the token on kubernetes pod at a specific location.

Reply:
I think we should not restrict the client within pod or need the deployment in OpenShift/kubernetes environment. Maybe we can explore how to get the token using service account.